pkg/api/db.go

```diff
@@ -240,7 +242,7 @@ func DeleteReplicationDBEntry(ctx *c.Context, in *model.ReplicationSpec) error {
 	invalidStatus := []string{model.ReplicationCreating, model.ReplicationDeleting, model.ReplicationEnabling,
 		model.ReplicationDisabling, model.ReplicationFailingOver, model.ReplicationFailingBack}
 
-	if utils.Contained(in.ReplicationStatus, invalidStatus) {
+	if utils.Contained(in.ReplicationStatus, validStatus) {
```

Comment:
I think you forgot to update this variable.

Reply:
OK，i am worried.